### ISSUE TITLE
How do we get the rom for ale examples?

### ISSUE THREAD

**Issue by arisliang** [chainer/chainerrl#268]
Do we need to download the rom and specify it ourselves? Do we have any examples?

**Comment by toslunar** [comment]
No, we don't have to.  `atari_py` contains rom data.  Could you try the following?
```
>>> import atari_py  # 0.1.1
>>> ale = atari_py.ALEInterface()
>>> ale.loadROM(atari_py.get_game_path('Adventure'))
>>> import matplotlib.pyplot as plt
>>> plt.imshow(ale.getScreenRGB())
<matplotlib.image.AxesImage object at 0x110f2dc88>
>>> plt.show()
```

**Comment by arisliang** [comment]
error at 3rd line `ale.loadROM(atari_py.get_game_path('Adventure'))`
> No ROM File specified or the ROM file was not found.

But I tried other game such as `python train_dqn_ale.py breakout` worked.

**Comment by arisliang** [comment]
The above lines works for lower case `adventure`, however training script `python train_dqn_ale.py adventure` doesn't work. It gives this error:
> This ROM is for PAL. Please use ROMs for NTSC

**Comment by toslunar** [comment]
> RuntimeError: This ROM is for PAL. Please use ROMs for NTSC

This error is on the screen size.  The default preprocessing supports only images with a certain size.

**Comment by muupan** [comment]
> RuntimeError: This ROM is for PAL. Please use ROMs for NTSC

Sorry, this is kind of a bug of the current `chainerrl.envs.ale.ALE`. Some of the ROMs in atari_py do not work because of this. To alleviate this problem, you can use your own NTSC ROM or use https://github.com/chainer/chainerrl/pull/253.

**Comment by muupan** [comment]
BTW it seems `gym` does not support Adventure while atari_py supports it.
